### PR TITLE
libgff: new package at version 2.0.0

### PR DESCRIPTION
**Pull request by alexander-baumgarten**

### CODE REVIEW

File: var/spack/repos/builtin/packages/libgff/package.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    maintainers = ['ajxander12']
+
+    version('2.0.0', sha256='7656b19459a7ca7d2fd0fcec4f2e0fd0deec1b4f39c703a114e8f4c22d82a99c')
+
+    # depends_on('foo')
+    depends_on('cmake@3.12:', type='build')
+
+    def cmake_args(self):
+        args = []
+        return args
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/libgff/package.py]: You can remove this function, since it's not necessary

**Review by alalazo — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/libgff/package.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    homepage = "https://github.com/COMBINE-lab/libgff/tree/v2.0.0"
+    url      = "https://github.com/COMBINE-lab/libgff/archive/v2.0.0.tar.gz"
+
+    # notify when the package is updated.
+    maintainers = ['ajxander12']
+
+    version('2.0.0', sha256='7656b19459a7ca7d2fd0fcec4f2e0fd0deec1b4f39c703a114e8f4c22d82a99c')
+
+    # depends_on('foo')
```
Comment (baberlevi) [inline var/spack/repos/builtin/packages/libgff/package.py]:
```suggestion
```

File: var/spack/repos/builtin/packages/libgff/package.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Libgff(CMakePackage):
+    """Lightweight GTF/GFF Parser, exposes a C++ interface without drawing in
+    a heavyweight dependency."""
+
+    homepage = "https://github.com/COMBINE-lab/libgff/tree/v2.0.0"
```
Comment (baberlevi) [inline var/spack/repos/builtin/packages/libgff/package.py]:
```suggestion
    homepage = "https://github.com/COMBINE-lab/libgff/
```

File: var/spack/repos/builtin/packages/libgff/package.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    url      = "https://github.com/COMBINE-lab/libgff/archive/v2.0.0.tar.gz"
+
+    # notify when the package is updated.
+    maintainers = ['ajxander12']
+
+    version('2.0.0', sha256='7656b19459a7ca7d2fd0fcec4f2e0fd0deec1b4f39c703a114e8f4c22d82a99c')
+
+    # depends_on('foo')
+    depends_on('cmake@3.12:', type='build')
```
Comment (baberlevi) [inline var/spack/repos/builtin/packages/libgff/package.py]: this is generally not required for CMakePackages - did you find it didn't work without this ?

**Review by baberlevi — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/libgff/package.py (hunk 4 above)
Comment (alexander-baumgarten) [reply var/spack/repos/builtin/packages/libgff/package.py]: Fixed.

**Review by alalazo — APPROVED** [review]

**Review by baberlevi — APPROVED** [review]

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**